src/pages/Login.js

```diff
@@ -1,12 +1,17 @@
 import { useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
 import { getRedirectResult } from 'firebase/auth';
 
```

Comment:
Extra space here. Let's remove it unless you did this intentionally. 
Let's add another ticket to have a fixed `imports` formatting with some standard.